.github/workflows/build.yml

```diff
@@ -84,7 +84,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x]
+        node-version: [18.x]
```

Comment:
please  rebase your branch on the master ;)
Theses changes are already done !

Reply:
Not sure if I've `rebased`. Was using github as my editor and I'm still a newb using `git`.